BranchSDK/src/BranchIO/Event/Event.h

```diff
@@ -10,6 +10,8 @@
 
 namespace BranchIO {
 
+namespace Event {
```

Comment:
What about 
```
namespace BranchIO::Event {
```

instead of nested namespaces?

Reply:
That's a C++17 extension. That basically doesn't exist in C++. This is how you do it.

Reply:
If this is the style you think best then ok.